README.md

```diff
@@ -1,4 +1,52 @@
 cf-k8s-networking
 ---
+​
+Routing and networking for Cloud Foundry running on Kubernetes.
 
-Don't try to use this yet!
+​
+![Architecture Diagram of CF-K8s-Networking](doc/assets/architecture.png)
```

Comment:
this image... isn't showing up

Reply:
It's weird -- sometimes when I first load the diff it is loading the image relative to the `master` branch and sometimes after I refresh the img source points to this branch and it works. This may just be an issue with Github?

![Screen Shot 2019-10-30 at 3 14 50 PM](https://user-images.githubusercontent.com/2096955/67903175-27698080-fb28-11e9-9c02-9e4c53e12dda.png)